### PR TITLE
plugins:puppetdb_foreman - Release 1.0.0

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: buildrepo [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (domcleal) [inline buildrepo]: Please remove these additional files.

File: rubygem-puppetdb_foreman/rubygem-puppetdb_foreman.spec [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 Summary:    Foreman plugin to interact with PuppetDB
 Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    0.2.0
+Version:    1.0.0
 Release:    2%{?foremandist}%{?dist}
```
Comment (domcleal) [inline rubygem-puppetdb_foreman/rubygem-puppetdb_foreman.spec]: reset to 1

**Comment by domcleal** [comment]
The minimum Foreman version should be updated to match engine.rb (I commented on https://github.com/theforeman/puppetdb_foreman/commit/480dd497625217d9eb91468837e07e417f18ec11 as I guess it should be 1.11, not 1.10).

**Comment by dLobatog** [comment]
Updated, thanks !

**Comment by domcleal** [comment]
Unfortunately the update to puppetdb_foreman's requirement isn't right still, it'll actually fail at runtime on 1.11.0 now: https://github.com/theforeman/puppetdb_foreman/commit/da50c6e4c324a7438c5c47c37fa44153ed45113b#commitcomment-16443921

<pre>
Foreman::PluginRequirementError: ERF72-9348 [Foreman::PluginRequirementError]: »puppetdb_foreman plugin requires Foreman > 1.11 but current is 1.11.0«
/home/dcleal/code/foreman/foreman/app/services/foreman/plugin.rb:130:in `requires_foreman'
/home/dcleal/.rvm/gems/ruby-2.0.0-p353@foreman/gems/puppetdb_foreman-1.0.1/lib/puppetdb_foreman/engine.rb:10:in `block (2 levels) in <class:Engine>'
</pre>

**Comment by domcleal** [comment]
Merged and building on both rpm/develop and rpm/1.11.  Thanks @dLobatog.

**Pull request merged**